### PR TITLE
refactor(device): Move demo-grove-pi contents to edgex-examples

### PR DESCRIPTION
**Pull request by BinduRao2018**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [NA ] Tests for the changes have been added (for bug fixes / features)
- [NA ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/Contributing.md.

## What is the current behavior?
demo-grovi-pi repository contain grove-c demo on Raspberry Pi

## What is the new behavior? 
Demo on Rpi 3b+ using  grove-c service added under device-services/grove-c

## Does this PR introduce a breaking change?

- [ ] Yes
- [X ] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: device-services/grove-c/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# demo-grove-pi
+Contain scripts to pull the images and run them for devkit based on Raspberry PI
+
+This repository contain:
```
Comment (jpwhitemn) [inline device-services/grove-c/README.md:4]: contain-> contains

File: device-services/grove-c/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# demo-grove-pi
+Contain scripts to pull the images and run them for devkit based on Raspberry PI
```
Comment (jpwhitemn) [inline device-services/grove-c/README.md:2]: Change to:  This example contains the instructions and scripts to connect the Grove Sensor Kit to a Raspberry Pi 3 and use EdgeX to get data from the kit's sensors and command the kit's devices.

This example uses the EdgeX Geneva release.

Comment (lenny-intel) [reply device-services/grove-c/README.md:2]: Should this example be upgraded to use Hanoi?

Comment (BinduRao2018) [reply device-services/grove-c/README.md:2]: > Should this example be upgraded to use Hanoi?

This revision of docker-compose is tested with Geneva release. There will be a separate PR with the required changes in to use Hanoi.

File: device-services/grove-c/README.md (hunk 1 above)
Comment (BinduRao2018) [reply device-services/grove-c/README.md]: Updated

**Review by jpwhitemn — APPROVED** [review]

**Pull request merged**